src/hooks/tests/protected-route.test.tsx

```diff
@@ -16,18 +16,20 @@ const resourceRoutes: ResourceRoutes = {
 }
 
 describe('protected routes ', () => {
-  it('routes for unauthorized user', () => {
-    const permissions = getPermissionsByRoles('user')
-    const { create, list, show, edit } = protectedRoutes(
-      permissions,
-      constants.R_ITEMS,
-      resourceRoutes
-    )
-    expect(list).toBeDefined()
-    expect(show).toBeDefined()
-    expect(create).toBeUndefined()
-    expect(edit).toBeUndefined()
-  })
+  // no longer required as unauthorized users can't see any data
+
```

Comment:
Hello @TahaKhanAbdalli - shouldn't we check these objects are _not_ defined?

Reply:
Since now there's no `user` role, therefore, we can't check these objects for any undefined role.

Reply:
Aah, that makes sense :-)